cmd/tailscale/cli/cli_test.go

```diff
@@ -493,7 +494,7 @@ func TestCheckForAccidentalSettingReverts(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			applyImplicitPrefs(newPrefs, tt.curPrefs, tt.curUser)
+			// applyImplicitPrefs(newPrefs, tt.curPrefs, tt.curUser)
```

Comment:
temporary debug?

Reply:
the function was removed and the functionality replaced... I need to remove the commend and function

Reply:
can you remove it in this PR? different commit is fine.